### ISSUE TITLE
Trouble adjusting length

### ISSUE THREAD

**Issue by sandoxy** [brarcher/video-transcoder#154]
It is diffucult to move the courser to the exact place, e.g. where the scene changes.

Thanks

**Comment by licaon-kter** [comment]
It has a "one second" precision afaics, a greater one would be nice,

**Comment by brarcher** [comment]
I agree that getting the cursor in just the right place is not easy. Hopefully the arrows on the side will help. If someone were interested in contributing changes to improve it I'd be glad to review the changes.

**Comment by sandoxy** [comment]
This app has a tool, which makes it easies to adjust the courser, because you see, when the scene changes. Maybe it helps:

https://gitlab.com/vijai/screenrecorder/issues
